app/views/payments/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_with(model: payment, url: payments_path(def_group: initial_group)) do |f| %>
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name, required: true, placeholder: 'Enter name' %>
+  </div>
+  <div class="field">
+    <%= f.label :amount %>
+    <%= f.number_field :amount, required: true, step: "0.01", placeholder: 'Enter amount' %>
+  </div>
+  <div class="field">
+    <%= f.label :categories %>
+    <% groups.each do |group| %>
+      <div>
+        <%= f.label "group_ids_#{group.id}" do %>
+          <%= f.check_box :group_ids, { multiple: true, checked: group.id == initial_group.id }, group.id, nil %>
+          <%= group.name %>
+        <% end %>
+      </div>
+    <% end %>
```

Comment:
@Lucas-Masaba, Thank you for your note. Is this part of the form acceptable as input for related groups?

Reply:
Hello @mohamedSabry0 , yes, you should add in the form field even if it is a related group as a project requirement.

Reply:
@Lucas-Masaba, 
Thank you for your time and continuous support.
One last clarification, I implemented the input for the group as a group of checkboxes. 
Is it required to be changed to a text field instead?

Reply:
@Lucas-Masaba,
Sorry for any confusion. I will wait for the next review, as it seems I had a problem in styling preventing them from appearing as expected.

Thank you again for your continuous support.